src/features/canvass/components/GLCanvassMap/index.tsx

```diff
@@ -171,6 +181,37 @@ const GLCanvassMap: FC<Props> = ({ assignment, selectedArea }) => {
     return locations.data?.find((loc) => loc.id == selectedLocationId) || null;
   }, [locations]);
 
+  const userLocationGeoJson: GeoJSON.FeatureCollection | null = useMemo(() => {
+    if (!userLocation) {
+      return null;
+    }
+
+    const zoom = mapZoom;
+    let accuracyPx = 18;
+
+    if (userAccuracy != null && zoom != null) {
+      const lat = userLocation[1];
+      const metersPerPixel =
+        (156543.03392 * Math.cos((lat * Math.PI) / 180)) / Math.pow(2, zoom);
```

Comment:
This is interesting math that I don't fully understand at a glance, but I don't feel like I have to because the `metersPerPixel` variable name already explains what the math does. 💯